x-pack/auditbeat/module/system/login/utmp.go

```diff
@@ -178,6 +177,7 @@ func (r *UtmpFileReader) readNewInFile(loginRecordC chan<- LoginRecord, errorC c
 	if !isKnownFile {
 		r.log.Debugf("Found new file: %v (utmpFile=%+v)", utmpFile.Path, utmpFile)
 	}
+	utmpFile.Offset = savedUtmpFile.Offset
 
 	size := utmpFile.Size
 	oldSize := savedUtmpFile.Size
```

Comment:
Down on line 214, it's missing to set the `err` value then proceeds to check the value of `err`.

Reply:
Ah, thanks. Added it.